src/components/buttons/CopyButton/CopyButton.scss

```diff
@@ -84,6 +99,7 @@
 
 .CopyButton__Text {
 	font-size: 12px;
+	letter-spacing: 0.24px;
```

Comment:
If this is a one off setting for this particular use case then ignore. If we use 0.24px elsewhere, can we use a scss variable or something to set this?

Reply:
You're correct, it's just the one-off in the code base. I'll remember that for future reference though!